src/styles/block/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+
+  &__list {
+    width: $widthCalendar;
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap;
+
+    @each $weekday, $number in $days {
+      &--start-day--#{$weekday} {
+        & :first-child {
+          margin-left: $number * ($ceilSize + $gap + 2px);
+        }
+      }
+    }
+
+    @for $day from 28 through 31 {
+      &--month-length--#{$day} {
+        & :nth-child(n + #{$day + 1}) {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__item {
+    box-sizing: inherit;
+    @include square($ceilSize);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    transition: transform 0.5s, background-color 0.5s;
+
+    @for $increment from 1 through 31 {
+      &--#{$increment}::before {
+        content: "#{$increment}";
+        color: black;
+      }
+    }
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: rgb(173, 54, 54);
```

Comment:
you can write the word 'pink' and the color will be like in the screenshot
![image](https://user-images.githubusercontent.com/110456763/186904241-1553596e-a9b9-4133-b58b-e27b12941124.png)

Reply:
You forgot about pointer
![image](https://user-images.githubusercontent.com/110456763/186905632-e7715ec5-8b9d-4b55-8090-3aa6afcffefe.png)